### DISCUSSION TITLE
Upcoming OpenDDS Version 3.25.0

### DISCUSSION THREAD

**Discussion by iguessthislldo** [OpenDDS/OpenDDS#4206] [Announcements]
The OpenDDS Foundation plans to release the master branch as OpenDDS 3.25.0 in the near future.  The current release notes can be found [here](https://opendds.readthedocs.io/en/master/news.html#v3-25-0). Any feedback would be greatly appreciated.

**Comment by jwillemsen** [comment]
AXCIOMA with OpenDDS master does seem to work as it should, see https://download.remedy.nl/buildlogs/rhel80_axcioma_master_opendds/ for the build/test results
